llvm/test/Transforms/InstCombine/win-fdim.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: opt %s -passes=instcombine -S -mtriple=i386-pc-mingw32 | FileCheck %s --check-prefixes=CHECK,MINGW32
+
+define double @fdim_double() {
+; CHECK-LABEL: define double @fdim_double() {
+; MINGW32:    ret double 2.500000e+00
+;
+  %dim = call double @fdim(double 10.5, double 8.0)
+  ret double %dim
+}
+
+define float @fdim_float() {
```

Comment:
@arsenm 
this test should fail because windows should not be able to fold @fdimf, right?

Reply:
Yes, the call should remain